.travis.yml

```diff
@@ -1,27 +1,27 @@
 jdk:
-  - oraclejdk8
```

Comment:
sorry for the reformatting of this file, travis did it... the env variables and any line with `client` in it is new

Reply:
https://github.com/broadinstitute/sam/pull/280/files?utf8=%E2%9C%93&diff=split&w=1 this link will make the whitespace differences in the GH view less painful